WaterdogPE/src/main/java/cn/stevei5mc/serverhelper/waterdogpe/listener/PlayerListener.java

```diff
@@ -29,7 +30,8 @@ public static void onPlayerChat(PlayerChatEvent event) {
     public static void onDispatchCommand(DispatchCommandEvent event) {
         if (main.getConfig().getBoolean("commands.usageLog.enable", true) && event.getSender().isPlayer()) {
             String cmd = event.getCommand().trim().toLowerCase();
-            ArrayList<String> secretsCmd = new ArrayList<>(main.getConfig().getStringList("commands.usageLog.secretsList"));
+            ArrayList<String> secretsCmd = Optional.ofNullable(main.getConfig().getStringList("commands.usageLog.secretsList"))
```

Comment:
虽然使用 Optional 可以避免空指针异常，但在这里直接使用 `getStringList` 的返回值应该已经足够安全，因为 Config 库通常会返回一个空列表而不是 null。

```suggestion
ArrayList<String> secretsCmd = new ArrayList<>(main.getConfig().getStringList("commands.usageLog.secretsList"));

Reply:
不这么干是真的会返回空指针的